src/test/java/org/example/action/BinaryAction.java

```diff
@@ -32,11 +33,16 @@
 @Status
 public class BinaryAction {
   public String expected;
+  public boolean expectedNullFile = false;
```

Comment:
Just as a coding standard, we generally don't initialize values to their default.

Reply:
I had a momentary brain freeze about how validation is supposed to work in Prime MVC. I had thought it was "built-in", didn't see any optional or required annotations out there. And now that I look again, I guess you do this explicitly with the `Validatable` interface plus the `@ValidationMethod` annotation. All that aside, it's also the default value too. Sorry about that.

Reply:
Fixed